nopolinfo #0
location: FailureLocation{className='org.postgresql.test.jdbc4.jdbc41.GetObjectTest', failingMethods=[org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetMoney], erroringMethods=[org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetSerial, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInet6Address, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInterval, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetDoubleNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBox, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetArray, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBigDecimal, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetCircle, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBooleanNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetTime, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetPolygon, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBoolean, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetPath, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetFloat, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBlob, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetTimestampWithTimeZone, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetDouble, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLongNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetPoint, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetIntegerNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLong, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetClob, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetDate, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLine, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInteger, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetTimestamp, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetString, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetXml, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInet4Address, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetFloatNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInetAddressNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLineseg, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetCalendar, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetUuid], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='skipped', failureDetail='skipped', isError=false}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=1, nbErrors=68}
status: NOPATCH
dateEnd: Mon Mar 20 01:49:07 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@3591009c', projectTests=[org.postgresql.test.jdbc4.jdbc41.GetObjectTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_ERRORING
----------



nopolinfo #1
location: FailureLocation{className='org.postgresql.test.jdbc4.jdbc41.GetObjectTest', failingMethods=[org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetMoney], erroringMethods=[org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetSerial, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInet6Address, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInterval, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetDoubleNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBox, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetArray, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBigDecimal, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetCircle, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBooleanNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetTime, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetPolygon, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBoolean, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetPath, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetFloat, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetBlob, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetTimestampWithTimeZone, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetDouble, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLongNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetPoint, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetIntegerNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLong, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetClob, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetDate, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLine, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInteger, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetTimestamp, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetString, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetXml, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInet4Address, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetFloatNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetInetAddressNull, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetLineseg, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetCalendar, org.postgresql.test.jdbc4.jdbc41.GetObjectTest#testGetUuid], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='skipped', failureDetail='skipped', isError=false}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=1, nbErrors=68}
status: NOPATCH
dateEnd: Mon Mar 20 01:49:20 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@21079a12', projectTests=[org.postgresql.test.jdbc4.jdbc41.GetObjectTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_FAILING
----------



nopolinfo #2
location: FailureLocation{className='org.postgresql.test.jdbc2.AutoRollbackTestSuite', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[32: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[44: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[72: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[30: autorollback(autoSave=NEVER, autoCommit=NO, failMode=INSERT_BATCH, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[93: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=INSERT_BATCH, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[18: autorollback(autoSave=NEVER, autoCommit=YES, failMode=INSERT_BATCH, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[8: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[47: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[70: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[61: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=INSERT_BATCH, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[3: autorollback(autoSave=NEVER, autoCommit=YES, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[26: autorollback(autoSave=NEVER, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[51: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[87: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[65: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[92: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[50: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=INSERT_BATCH, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[41: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[64: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[14: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[24: autorollback(autoSave=NEVER, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[86: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[1: autorollback(autoSave=NEVER, autoCommit=YES, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[82: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=INSERT_BATCH, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[85: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[62: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=INSERT_BATCH, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[23: autorollback(autoSave=NEVER, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[88: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[25: autorollback(autoSave=NEVER, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[54: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[20: autorollback(autoSave=NEVER, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[10: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[57: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[90: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[31: autorollback(autoSave=NEVER, autoCommit=NO, failMode=INSERT_BATCH, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[34: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[36: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[73: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[71: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[74: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[29: autorollback(autoSave=NEVER, autoCommit=NO, failMode=INSERT_BATCH, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[11: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[81: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=INSERT_BATCH, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[94: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=INSERT_BATCH, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[0: autorollback(autoSave=NEVER, autoCommit=YES, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[35: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[69: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[52: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[75: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[40: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[22: autorollback(autoSave=NEVER, autoCommit=NO, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[48: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=INSERT_BATCH, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[39: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[91: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[37: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[49: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=INSERT_BATCH, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[58: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[33: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[78: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[2: autorollback(autoSave=NEVER, autoCommit=YES, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[38: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[27: autorollback(autoSave=NEVER, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[13: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[63: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=INSERT_BATCH, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[17: autorollback(autoSave=NEVER, autoCommit=YES, failMode=INSERT_BATCH, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[16: autorollback(autoSave=NEVER, autoCommit=YES, failMode=INSERT_BATCH, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[46: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[83: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[6: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[66: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[76: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[68: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[80: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=INSERT_BATCH, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[42: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[45: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[60: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[19: autorollback(autoSave=NEVER, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[95: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=INSERT_BATCH, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[7: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[12: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[15: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[84: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[21: autorollback(autoSave=NEVER, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[55: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[59: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[28: autorollback(autoSave=NEVER, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[89: autorollback(autoSave=CONSERVATIVE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[67: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[4: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[56: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[77: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[9: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[79: autorollback(autoSave=CONSERVATIVE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[43: autorollback(autoSave=ALWAYS, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[5: autorollback(autoSave=NEVER, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false)], org.postgresql.test.jdbc2.AutoRollbackTestSuite#run[53: autorollback(autoSave=ALWAYS, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true)]], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=96}
status: NOPATCH
dateEnd: Mon Mar 20 01:49:31 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@5792c08c', projectTests=[org.postgresql.test.jdbc2.AutoRollbackTestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 1
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #3
location: FailureLocation{className='org.postgresql.test.jdbc2.optional.PoolingDataSourceTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testDdlOverConnection, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testGetConnection, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testDoubleConnectionClose, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testJndi, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testCantReuseName, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testUseConnection, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testPGConnection, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testNotPooledConnection, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testDoubleStatementClose, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testConnectionObjectMethods, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testCreateDataSource, org.postgresql.test.jdbc2.optional.PoolingDataSourceTest#testStatementObjectMethods], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=24}
status: NOPATCH
dateEnd: Mon Mar 20 01:49:42 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@76216830', projectTests=[org.postgresql.test.jdbc2.optional.PoolingDataSourceTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #4
location: FailureLocation{className='org.postgresql.test.jdbc3.Jdbc3BlobTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.Jdbc3BlobTest#testAllBytes, org.postgresql.test.jdbc3.Jdbc3BlobTest#test1ByteOffset, org.postgresql.test.jdbc3.Jdbc3BlobTest#test1ByteStream, org.postgresql.test.jdbc3.Jdbc3BlobTest#testManyBytesOffsetStream, org.postgresql.test.jdbc3.Jdbc3BlobTest#testAllBytesStream, org.postgresql.test.jdbc3.Jdbc3BlobTest#test1ByteOffsetStream, org.postgresql.test.jdbc3.Jdbc3BlobTest#testManyBytesOffset, org.postgresql.test.jdbc3.Jdbc3BlobTest#testTruncate, org.postgresql.test.jdbc3.Jdbc3BlobTest#testPattern, org.postgresql.test.jdbc3.Jdbc3BlobTest#testManyBytesStream, org.postgresql.test.jdbc3.Jdbc3BlobTest#testManyBytes, org.postgresql.test.jdbc3.Jdbc3BlobTest#test1Byte], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=24}
status: NOPATCH
dateEnd: Mon Mar 20 01:49:51 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@1cc41b77', projectTests=[org.postgresql.test.jdbc3.Jdbc3BlobTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #5
location: FailureLocation{className='org.postgresql.test.jdbc2.optional.ConnectionPoolTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testJndi, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testDdlOverConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testPoolReuse, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testBackendIsClosed, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testCloseEvent, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testPoolCloseOldWrapper, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testPreparedStatementConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testGetConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testCreateDataSource, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testStatementsProxyPGStatement, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testNoCloseEvent, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testUseConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testNotPooledConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testPoolNewWrapper, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testAutomaticCloseEvent, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testStatementConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testIsClosed, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testCallableStatementConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testInlineCloseEvent, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testSerializable, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testPGConnection, org.postgresql.test.jdbc2.optional.ConnectionPoolTest#testStatementProxy], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=22}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:00 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@1ef93e01', projectTests=[org.postgresql.test.jdbc2.optional.ConnectionPoolTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #6
location: FailureLocation{className='org.postgresql.test.jdbc3.Jdbc3SavepointTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.Jdbc3SavepointTest#testRollingBackToInvalidSavepointFails, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testContinueAfterError, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testReleaseSavepoint, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testCantMixSavepointTypes, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testComplicatedSavepointName, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testRollingBackToSavepoints, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testRollbackMultipleTimes, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testGlobalRollbackWorks, org.postgresql.test.jdbc3.Jdbc3SavepointTest#testAutoCommitFails], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=18}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:10 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@37753b69', projectTests=[org.postgresql.test.jdbc3.Jdbc3SavepointTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #7
location: FailureLocation{className='org.postgresql.test.jdbc42.SetObject310Test', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc42.SetObject310Test#testSetLocalDateTime, org.postgresql.test.jdbc42.SetObject310Test#testSetLocalTimeWithType, org.postgresql.test.jdbc42.SetObject310Test#testSetOffsetDateTime, org.postgresql.test.jdbc42.SetObject310Test#testSetLocalDateWithoutType, org.postgresql.test.jdbc42.SetObject310Test#testSetLocalTimeWithoutType, org.postgresql.test.jdbc42.SetObject310Test#testSetLocalDateWithType, org.postgresql.test.jdbc42.SetObject310Test#testSetLocalTimeAndReadBack, org.postgresql.test.jdbc42.SetObject310Test#testSetLocalDateTimeBc], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=16}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:18 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@10b2c1cf', projectTests=[org.postgresql.test.jdbc42.SetObject310Test], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #8
location: FailureLocation{className='org.postgresql.test.jdbc4.jdbc41.SchemaTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc4.jdbc41.SchemaTest#testSearchPathPreparedStatementAutoCommitFalse, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testMultipleSearchPath, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testSchemaInProperties, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testSearchPathPreparedStatementAutoCommitTrue, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testGetSetSchema, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testSchemaPath$User, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testSearchPathPreparedStatement, org.postgresql.test.jdbc4.jdbc41.SchemaTest#testUsingSchema], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=16}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:24 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@23f539df', projectTests=[org.postgresql.test.jdbc4.jdbc41.SchemaTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #9
location: FailureLocation{className='org.postgresql.test.jdbc2.optional.SimpleDataSourceTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testNotPooledConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testPGConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testDdlOverConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testUseConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testGetConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testJndi, org.postgresql.test.jdbc2.optional.SimpleDataSourceTest#testCreateDataSource], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=14}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:31 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@53813f01', projectTests=[org.postgresql.test.jdbc2.optional.SimpleDataSourceTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #10
location: FailureLocation{className='org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testCreateDataSource, org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testPGConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testUseConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testDdlOverConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testJndi, org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testNotPooledConnection, org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest#testGetConnection], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=14}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:37 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@47a72c32', projectTests=[org.postgresql.test.jdbc2.optional.SimpleDataSourceWithUrlTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #11
location: FailureLocation{className='org.postgresql.test.jdbc4.jdbc41.CloseOnCompletionTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc4.jdbc41.CloseOnCompletionTest#testMultipleResultSet, org.postgresql.test.jdbc4.jdbc41.CloseOnCompletionTest#testNoResultSet, org.postgresql.test.jdbc4.jdbc41.CloseOnCompletionTest#testSingleResultSet, org.postgresql.test.jdbc4.jdbc41.CloseOnCompletionTest#testWithoutCloseOnCompletion], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:44 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@bf0025d', projectTests=[org.postgresql.test.jdbc4.jdbc41.CloseOnCompletionTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #12
location: FailureLocation{className='org.postgresql.test.jdbc3.TypesTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.TypesTest#testPreparedByte, org.postgresql.test.jdbc3.TypesTest#testUnknownType, org.postgresql.test.jdbc3.TypesTest#testPreparedBoolean, org.postgresql.test.jdbc3.TypesTest#testCallableBoolean], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:50 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@64d49e35', projectTests=[org.postgresql.test.jdbc3.TypesTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #13
location: FailureLocation{className='org.postgresql.test.jdbc3.ParameterMetaDataTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.ParameterMetaDataTest#testParameterMD, org.postgresql.test.jdbc3.ParameterMetaDataTest#testTypeChangeWithUnknown, org.postgresql.test.jdbc3.ParameterMetaDataTest#testMultiStatement, org.postgresql.test.jdbc3.ParameterMetaDataTest#testFailsOnBadIndex], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Mon Mar 20 01:50:55 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@42fa1554', projectTests=[org.postgresql.test.jdbc3.ParameterMetaDataTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #14
location: FailureLocation{className='org.postgresql.test.jdbc42.GetObject310Test', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc42.GetObject310Test#testGetLocalTimeInvalidType, org.postgresql.test.jdbc42.GetObject310Test#testGetLocalTimeNull, org.postgresql.test.jdbc42.GetObject310Test#testGetLocalDateTime, org.postgresql.test.jdbc42.GetObject310Test#testGetLocalTime, org.postgresql.test.jdbc42.GetObject310Test#testGetTimestampWithTimeZone, org.postgresql.test.jdbc42.GetObject310Test#testGetLocalDate], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=6}
status: NOPATCH
dateEnd: Mon Mar 20 01:51:07 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@7addc5e4', projectTests=[org.postgresql.test.jdbc42.GetObject310Test], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 7
nbAngelicValues: 6
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #15
location: FailureLocation{className='org.postgresql.test.jdbc42.GetObject310BinaryTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc42.GetObject310BinaryTest#testGetLocalTimeInvalidType, org.postgresql.test.jdbc42.GetObject310BinaryTest#testGetLocalDateTime, org.postgresql.test.jdbc42.GetObject310BinaryTest#testGetTimestampWithTimeZone, org.postgresql.test.jdbc42.GetObject310BinaryTest#testGetLocalTime, org.postgresql.test.jdbc42.GetObject310BinaryTest#testGetLocalTimeNull, org.postgresql.test.jdbc42.GetObject310BinaryTest#testGetLocalDate], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=6}
status: NOPATCH
dateEnd: Mon Mar 20 01:51:22 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@61be2a65', projectTests=[org.postgresql.test.jdbc42.GetObject310BinaryTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 15
nbAngelicValues: 12
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #16
location: FailureLocation{className='org.postgresql.test.jdbc3.DatabaseMetaDataTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.DatabaseMetaDataTest#testGetColumnsForDomain], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:51:28 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@5d6bed7b', projectTests=[org.postgresql.test.jdbc3.DatabaseMetaDataTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #17
location: FailureLocation{className='org.postgresql.test.jdbc42.CustomizeDefaultFetchSizeTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc42.CustomizeDefaultFetchSizeTest#testSetPredefineDefaultFetchSizeOnPreparedStatement, org.postgresql.test.jdbc42.CustomizeDefaultFetchSizeTest#testSetPredefineDefaultFetchSizeOnStatement], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:51:40 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@449ca6b6', projectTests=[org.postgresql.test.jdbc42.CustomizeDefaultFetchSizeTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 8
nbAngelicValues: 6
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #18
location: FailureLocation{className='org.postgresql.test.jdbc4.jdbc41.AbortTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc4.jdbc41.AbortTest#testAbortOnClosedConnection, org.postgresql.test.jdbc4.jdbc41.AbortTest#testAbort], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:51:45 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@2b360641', projectTests=[org.postgresql.test.jdbc4.jdbc41.AbortTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #19
location: FailureLocation{className='org.postgresql.test.jdbc42.PreparedStatementTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc42.PreparedStatementTest#testTimeTzSetNull, org.postgresql.test.jdbc42.PreparedStatementTest#testTimestampTzSetNull], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:51:56 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@752448d2', projectTests=[org.postgresql.test.jdbc42.PreparedStatementTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 7
nbAngelicValues: 6
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #20
location: FailureLocation{className='org.postgresql.test.jdbc3.ResultSetTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.ResultSetTest#testHoldableResultSet], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:52:03 CET 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@4bd23b22', projectTests=[org.postgresql.test.jdbc3.ResultSetTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #21
location: FailureLocation{className='org.postgresql.test.jdbc2.optional.CaseOptimiserDataSourceTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.optional.CaseOptimiserDataSourceTest#testDataSourceDisabledSanitiserPropertySucceeds], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:53:34 CET 2017
allocatedtime: 120minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@2f63ef53', projectTests=[org.postgresql.test.jdbc2.optional.CaseOptimiserDataSourceTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 130
nbAngelicValues: 99
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #22
location: FailureLocation{className='org.postgresql.test.jdbc3.SendRecvBufferSizeTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.SendRecvBufferSizeTest#testSelect], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}, FailureType{failureName='java.lang.NullPointerException:', failureDetail='null', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Mar 20 01:53:39 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@2700252f', projectTests=[org.postgresql.test.jdbc3.SendRecvBufferSizeTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #23
location: FailureLocation{className='org.postgresql.test.jdbc4.Jdbc4TestSuite', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc4.Jdbc4TestSuite#initializationError], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:53:45 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@7e5ed9ce', projectTests=[org.postgresql.test.jdbc4.Jdbc4TestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #24
location: FailureLocation{className='org.postgresql.test.jdbc3.StringTypeParameterTest', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc3.StringTypeParameterTest#testParameterUnspecified], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:53:52 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@354d078f', projectTests=[org.postgresql.test.jdbc3.StringTypeParameterTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #25
location: FailureLocation{className='org.postgresql.test.xa.XATestSuite', failingMethods=[], erroringMethods=[org.postgresql.test.xa.XATestSuite#initializationError], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:53:58 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@7eba6729', projectTests=[org.postgresql.test.xa.XATestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #26
location: FailureLocation{className='org.postgresql.test.jdbc42.SimpleJdbc42Test', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc42.SimpleJdbc42Test#testSupportsRefCursors], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:54:03 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@2c4386a6', projectTests=[org.postgresql.test.jdbc42.SimpleJdbc42Test], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #27
location: FailureLocation{className='org.postgresql.replication.ReplicationTestSuite', failingMethods=[], erroringMethods=[org.postgresql.replication.ReplicationTestSuite#org.postgresql.replication.ReplicationTestSuite], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:54:09 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@788577d3', projectTests=[org.postgresql.replication.ReplicationTestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #28
location: FailureLocation{className='org.postgresql.test.socketfactory.SocketFactoryTestSuite', failingMethods=[], erroringMethods=[org.postgresql.test.socketfactory.SocketFactoryTestSuite#testDatabaseMetaData], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:54:19 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@68154374', projectTests=[org.postgresql.test.socketfactory.SocketFactoryTestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 8
nbAngelicValues: 5
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #29
location: FailureLocation{className='org.postgresql.test.extensions.ExtensionsTestSuite', failingMethods=[], erroringMethods=[org.postgresql.test.extensions.ExtensionsTestSuite#initializationError], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:54:25 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@42fe8be4', projectTests=[org.postgresql.test.extensions.ExtensionsTestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #30
location: FailureLocation{className='org.postgresql.test.jdbc2.Jdbc2TestSuite', failingMethods=[], erroringMethods=[org.postgresql.test.jdbc2.Jdbc2TestSuite#initializationError], failures=[FailureType{failureName='org.postgresql.util.PSQLException', failureDetail='Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Mar 20 01:54:31 CET 2017
allocatedtime: 119minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/pgjdbc/pgjdbc/212757009/pgjdbc/src/main/java], projectClasspath='[Ljava.net.URL;@34246777', projectTests=[org.postgresql.test.jdbc2.Jdbc2TestSuite], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



